components/41-organisms/timeline/_timeline.scss

```diff
@@ -238,7 +238,7 @@ dl.timeline {
 
       @include desktop {
         margin: 0;
-        padding: 0 calc(#{$gutter-width/2}) 0 calc(#{$timeline-dt-left-padding} + #{$timeline-dt-right-padding});
+        padding: 0 calc(#{$gutter-width*.5}) 0 calc(#{$timeline-dt-left-padding} + #{$timeline-dt-right-padding});
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.